### PR TITLE
docs: improve `/tutorials/quick-start` section.

### PR DESCRIPTION
**Pull request by destifo**
- [x] Improve Layout
- [x] Include a simple project.
- [x] Add the result/outputs to running CLI commands.
- [x] Remove Metatype cloud registration form.
- [x] Separate the CLI commands to separate code blocks
- [x] Add links to references and concepts.
- [x] Add playground.

#### Motivation and context

[Docs Meta-task](https://linear.app/metatypedev/issue/MET-440/docs-meta-task)

#### Migration notes

_No Migration Needed_

### Checklist

- [x] Test the commands and the examples.
- [ ] The change come with new or modified tests

### CODE REVIEW

File: website/docs/tutorials/quick-start/index.mdx [hunk 1]
```diff
@@ -38,7 +41,41 @@ The doctor subcommand will attempt to detect all the components and report any p
...
 meta doctor
 ```
-    TODO: Add result image/log
+After Sucessful installation, the above command produces an output somewhat similar to the one below.
+```console
+user@first-project:~$ meta doctor
+——————————————————————————— Global ———————————————————————————
+curr. directory      /Users/user/Documents/metatype-playground/projects/first-project
+global config        /Users/user/Library/Application Support/dev.metatype.meta/config.json
+meta-cli version     0.3.6
```
Comment (Natoandro) [inline website/docs/tutorials/quick-start/index.mdx:50]: Could we link this to the published version in dev/lock.yml?

Comment (destifo) [reply website/docs/tutorials/quick-start/index.mdx:50]: Ok, but I did `meta upgrade` and `0.3.6` was the latest one.

**Review by Yohe-Am — APPROVED** [review]

File: website/docs/tutorials/quick-start/index.mdx [hunk 2]
```diff
@@ -63,3 +102,7 @@ ext install metatypedev.vscode-metatype
 Alternatively, you can download the `.vsix` file from the
 [releases page](https://github.com/metatypedev/metatype/releases)
 and manually install the extension.
+
+## Writing your First Project
```
[the remark this answers is not in the document]
Comment (destifo) [reply website/docs/tutorials/quick-start/index.mdx:106]: noted!

[the remark this answers is not in the document]
Comment (destifo) [reply website/docs/tutorials/quick-start/index.mdx:106]: Done.

**Review by Yohe-Am — DISMISSED** [review]

File: website/shared/projects/tab-first-project-python.mdx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+# install dependencies
+poetry install
+
+# create and activate virtual environment
+poetry shell
+```
+
+This will create the necessary files for development, some of which are:
+- `.graphqlrc.yaml`: configuration file to define settings and options related to.
```
[the remark this answers is not in the document]
Comment (destifo) [reply website/shared/projects/tab-first-project-python.mdx:19]: done.

**Review by Yohe-Am — APPROVED** [review]

**Pull request merged**